### ISSUE TITLE
BarThickness and MaxBarThickness

### ISSUE THREAD

**Issue by johnnyleroux** [Megabit/Blazorise#2547]
I cannot seem to get BarThickness or MaxBarThickness to work

I specify it like this:

```
  ChartOptions = new BarChartOptions
        {
            BarThickness = 5,
            Scales = new Scales
            {
                XAxes = new List<Axis>
                    {
                    new Axis
                        {
                        Stacked = true
                        }
                    },
                YAxes = new List<Axis>
                    {
                    new Axis
                        {
                        Stacked = true
                        }
                    }
            }
        };
```

Stacking works correctly, so the options are being applied. I've tried with various values for BarThickness and MaxBarThickness (separately) and nothing seems to make a difference.

**Comment by stsrki** [comment]
According to this [answer on stackoverflow](https://stackoverflow.com/questions/48906453/how-to-set-width-of-bar-dynamically-in-chart-js) it seems chart documentation is wrong or misleading.

`BarThickness` should be on the `Axis`. I will fix it for the 0.9.4 release.

**Comment by johnnyleroux** [comment]
Working perfectly now. Thank you very much!
